portfolio/src/main/java/com/google/sps/servlets/DeleteServlet.java

```diff
@@ -0,0 +1,26 @@
+import java.io.IOException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import com.google.appengine.api.datastore.PreparedQuery;
+import com.google.appengine.api.datastore.Query;
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
+
+@WebServlet("/delete-data")
+public class DeleteServlet extends HttpServlet {
+   @Override
+  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
+    Query query = new Query("Comment");
```

Comment:
The string "Comment" is now used across multiple files. A good practice is creating a separate file exporting the constant and importing everywhere where it is used. This makes any future refactors that change "Comment" sting to be a lot easier as you will only have to change it in one place.

Reply:
I would go even farther and suggest creating a class that contains all database logic for comments (ex. Add, Query, Delete) and that is then used by the serving code.